### DISCUSSION TITLE
Unable to submit a bug report

### DISCUSSION THREAD

**Discussion by MaxBear** [netbox-community/netbox#15916] [General]
![NetboxIssue](https://github.com/netbox-community/netbox/assets/1416049/06f2cc20-2ff3-4e03-acea-d68d187f8bd9)

**Comment by jeremystretch** [comment]
The NetBox version field is required. The gray text you see is an example for reference.

Also FYI, slim-select has been replaced with tom-select in the upcoming v4.0 release.

**Reply by MaxBear** [reply]
Thanks, i though that field already has a value populated

**Reply by bitcollector1** [reply]
This same exact thing happened to me awhile back after a very long day I did not notice the value was not entered into the box.
